style.css

```diff
@@ -1,3 +1,136 @@
+
```

Comment:
- [ ] As mentioned by the previous reviewers, kindly note that your mobile version should still remain intact even after adding a media query for the desktop version, this is so that your app can be responsive over different screen sizes. All CSS that belongs to only the desktop version should be in the media query so it doesn't affect the mobile version.
### Your mobile design
![1](https://user-images.githubusercontent.com/94551484/213886159-ec4edfab-5d68-45d6-8e4e-085a1d9741e6.png)
![2](https://user-images.githubusercontent.com/94551484/213886162-bba8ddc1-e74c-4bd7-87e8-1108edc1cd53.png)
![3](https://user-images.githubusercontent.com/94551484/213886163-63c89f4d-b6e1-470d-a427-0dec3b47bc6f.png)
### Figma designs
![f1](https://user-images.githubusercontent.com/94551484/213886166-ecbd3551-8845-474c-9b0b-72d08e3274bc.png)
![f2](https://user-images.githubusercontent.com/94551484/213886169-8b2ecc12-ab7a-48cd-a4f9-cee9c442d3bf.png)
![f3](https://user-images.githubusercontent.com/94551484/213886170-6c174bb7-fc8a-497e-b0de-8e3941a260e5.png)